board/views.py

```diff
@@ -1,23 +1,27 @@
+from uuid import UUID
+from django.core import exceptions
 from django.http import HttpResponse, JsonResponse
+from django.http.response import Http404
 from django.views import View
 from board.forms import PostForm
```

Comment:
A space should be between the builtin and library imports

Reply:
Also library imports should be ordered alphabetically

Reply:
👍